autoreject/autoreject.py

```diff
@@ -50,7 +50,7 @@ def validation_curve(epochs, y=None, param_name="thresh", param_range=None,
     ----------
     epochs : instance of mne.Epochs.
         The epochs.
-    y : array | None
+    y : array | None
```

Comment:
That was an auto-modification by my IDE, I didn't pay attention. I don't see the change, do you?

Reply:
Nopes, it's fine. You can leave it as is